### ISSUE TITLE
Error in query.cont[query.use] : invalid subscript type 'list'

### ISSUE THREAD

**Issue by mproctor340** [16EAGLE/getSpatialData#59]
Getting this error (intermittently I think) when running getLandsat_query. I've traced it back to .EE_query in Internal.R

Here is the code I use:

```R
  # Set start and end date for cloud cover (as far back as GLAD alerts go to current day)
  start_date <- paste0(year-num_years, "-01-01")
  end_date <- as.character(Sys.Date())
  
  # Query USGS database for all Landsat scene metadata in our area
  query <- getLandsat_query(c(start_date, end_date),
                            name="all",
                            aoi=remove.holes(aoi_sp),
                            verbose=TRUE
  )
```

**Comment by 16EAGLE** [comment]
Hi, could you retry with the latest major release, whether this error has been resolved? Please note that version 0.1.0 introduces some UI changes that might break some old code, however most functions should still work. See the changes here: http://jxsw.de/getSpatialData/news/index.html#getspatialdata-0-1-0

**Comment by 16EAGLE** [comment]
closing due to inactivity. Please reopen if needed!
